test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -25,22 +25,30 @@
 IGNORE_SAFETY_IDS = {
     "tensorflow": {
         "training": {
-            "py2": ['37524'],
+            "py2": ['37524',
```

Comment:
Could you remove 37524 from all the lists?

Reply:
Removed. Thanks